src/internet_identity/src/storage/registration_rates.rs

```diff
@@ -112,8 +112,6 @@ fn calculate_registration_rate<M: Memory>(now: u64, data: &MinHeap<Timestamp, M>
         .peek()
         // calculate the time window length with respect to the current time
         .map(|ts| now - ts)
-        // There could be negative values if they haven't been pruned yet.
-        .filter(|val| *val != 0)
```

Comment:
No,no you still need the 0 check. It's just the comment that was wrong. I.e. it should say filter out 0 rather than talk about negative values.

Reply:
oops, yes. Good that there was a test for it